### DISCUSSION TITLE
user-guide/faqs/caj2pdf

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#321] [文档评论区]
# user-guide/faqs/caj2pdf

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/faqs/caj2pdf

**Comment by amommm** [comment]
WPS的pdf编辑器也可以

**Reply by Chikit-L** [reply]
我这里测试了一下发现最新的WPS office还是无法打开caj文件的，请问你这里使用的是哪个版本呢

**Reply by guolizhang-tyler** [reply]
wps可以caj转pdf，不过逻辑好像是把caj转换成图片型pdf，不是很方便
